cmd/kubeadm/app/preflight/checks.go

```diff
@@ -607,3 +607,20 @@ func TryStartKubelet() {
 		}
 	}
 }
+
+// RunPreFlightChecks runs all pre-flight checks
+func RunPreFlightChecks(cfg *kubeadmapi.MasterConfiguration) error {
+	fmt.Println("[preflight] Running pre-flight checks")
```

Comment:
Makes sense! After all, if someone invoked `alpha phase preflight` they do know the pre-flight checks are running (as opposed to not knowing all the phases that `init` comprises.